README.md

```diff
@@ -0,0 +1,140 @@
+#<a name="top"></a>FIWARE Testbed Deploy
+
+[![License badge](https://img.shields.io/badge/license-Apache_2.0-blue.svg)](LICENSE)
+[![Documentation badge](https://readthedocs.org/projects/fiware-testbed-deploy/badge/?version=latest)](http://fiware-testbed-deploy.readthedocs.org/en/latest/?badge=latest)
+[![Docker badge](https://img.shields.io/docker/pulls/fiware/fiware-testbed-deploy.svg)](https://hub.docker.com/r/fiware/fiware-deploy-one-testbed/)
+[![Support badge]( https://img.shields.io/badge/support-sof-yellowgreen.svg)](http://stackoverflow.com/questions/tagged/fiware-testbed-deploy)
+
+* [Introduction](#introduction)
+* [Overall description](#overall-description)
+* [Build and Install](#build-and-install)
+* [API Overview](#api-overview)
+* [Testing](#testing)
+    * [Unit Tests](#unit-tests)
+    * [End-to-end tests](#end-to-end-tests)
+* [Advanced topics](#advanced-topics)
+* [Support](#support)
+* [License](#license)
+
+
+## Introduction
+
+This is the code repository for **FIWARE Testbed Deploy component**. This project is a scripts sets developed to deploy Openstack testbeds
+
+This project is part of [FIWARE](http://www.fiware.org).
+
+Any feedback on this documentation is highly welcome, including bugs, typos
+or things you think should be included but aren't. You can use [github issues](https://github.com/telefonicaid/fiware-testbed-deploy/issues/new) to provide feedback.
+
+[Top](#top)
+
+## Overall description
+A set of scripts to create a secure environment for testing is provided. The
+environment is created in a virtual machine using the FIWARE Lab infrastructure.
+The environment variables OS_AUTH_URL, OS_REGION_NAME, OS_USERNAME, OS_PASSWORD
+and OS_TENANT_NAME must be filled with the data of a valid FIWARE Lab account.
+
+The installation of the testbed is fully automatized and consists on an OpenStack
+instance where the original keystone server was replaced with the KeyRock server.
+This is work in progress; the current version only installs Glance, Nova and Neutron,
+but skuld also purges Swift, Cinder and Blueprint resources.
+
+### Deploying one testbed
+By using a fiware-testbed-deploy script (*launch_vm.py*), it is possible to deploy an entire Openstack testbed.
+The script uses a floating IP and creates both a keypair (it saves the SSH private key as
+~/.ssh/testbedskuld_key) and a security group. The security group opens
+the port 22 (SSH) and allows the ICMP traffic. It is also possible to connect
+to any port from a VM using the same security group.
+
+The names of the keypair, the security group and other parameters as the preferable
+floating IP are configurable in the *settings.py* file, but most of the users may
+ignore this file securely. However, to use the script in a different platform
+than FIWARE Lab, probably is necessary to change the parameter about the
+public shared network name.
```

Comment:
You do not need to change the references to the Keystone API and rest of OpenStack services when you run it on a different environment?